scipy/_lib/decorator.py

```diff
@@ -42,30 +42,7 @@
 
 __version__ = '4.0.5'
 
-if sys.version >= '3':
```

Comment:
I'm ambivalent on making changes to this file. It's a vendored piece of code and I'm not sure the reward for making changes is big enough. If any changes to the original code are made it's harder to include them here.

Reply:
reverted.